src/lib.rs

```diff
@@ -20,18 +21,17 @@ mod near_backend;
 #[cfg(feature = "contract")]
 mod sdk;
 
-use evm_core::ToStr;
-
 #[cfg(feature = "contract")]
 mod contract {
     use borsh::BorshDeserialize;
+    use primitive_types::{H160, H256};
 
+    use crate::evm_core::ExitReason;
     use crate::near_backend::Backend;
+    use crate::runtime::ToStr;
```

Comment:
Just side note, that `ToStr` should be replaced with std's `ToString` trait and then referenced from that String.

Reply:
given this compiles in both no_std and std, it's sometimes easier to have a short cut. Not sure if this is the one though.

Reply:
As Illia noted, we want to compile this with `no_std`. Hence, `ToStr` is the right call here. (Added in https://github.com/near/near-evm/commit/59007343dd87c00be116e58800b7a547b35da453.)